RN 43457 (#5497)
SkylineCommunications/dataminer-docs: dataminer/Reference/Skyline_DataMiner_Folder/More_information_on_certain_files_and_folders/ClientSettings_json.md

}
```

From DataMiner 10.4.0 [CU21]/10.5.0 [CU9]/10.5.12 onwards<!--RN 43457-->, it is possible to cache specific columns of a table using the optional `ColumnIDs` property. If no `ColumnIDs` property is specified, the entire table will be cached.

For example, the following configuration shows how to cache only specific columns of a table parameter:

```json

{
    "DataMinerID": 956,
    "ElementID": 1846,
    "ParameterID": 1500,
    "SubscriptionFilter": null,
    "ColumnIDs": "1512"
}
```

> [!NOTE]
>
> - A `columns=` option can also be provided in the `SubscriptionFilter`. In this case, it takes priority over the separate `ColumnIDs` setting.

> > - When `columns=` is used in `SubscriptionFilter`, only requests that use the same `SubscriptionFilter` will use the cache. For the separate `ColumnIDs` setting, any request that matches the other cache settings will use the cached data if the requested column matches the cached column.

Author: @KlaasSL, does it look OK now?

> > - The `ColumnIDs=` parameter can also be provided in the `SubscriptionFilter`. In this case, it takes priority over the separate `ColumnIDs` setting. When `ColumnIDs=` is used, only requests that use the same `SubscriptionFilter` will use the cache. For the separate `ColumnIDs` setting, any request that matches the other cache settings will use the cached data if the requested column matches the cached column.

Author: @KlaasSL, so I adjust it like this?
```suggestion
> - A `columns=` option can also be provided in the `SubscriptionFilter`. In this case, it takes priority over the separate `ColumnIDs` setting. When `columns=` is used in `SubscriptionFilter`, only requests that use the same `SubscriptionFilter` will use the cache. For the separate `ColumnIDs` setting, any request that matches the other cache settings will use the cached data if the requested column matches the cached column.
```

> > - A `columns=` option can also be provided in the `SubscriptionFilter`. In this case, it takes priority over the separate `ColumnIDs` setting. When `columns=` is used in `SubscriptionFilter`, only requests that use the same `SubscriptionFilter` will use the cache. For the separate `ColumnIDs` setting, any request that matches the other cache settings will use the cached data if the requested column matches the cached column.

Author: ```suggestion
> - A `columns=` option can also be provided in the `SubscriptionFilter`. In this case, it takes priority over the separate `ColumnIDs` setting.
> - When `columns=` is used in `SubscriptionFilter`, only requests that use the same `SubscriptionFilter` will use the cache. For the separate `ColumnIDs` setting, any request that matches the other cache settings will use the cached data if the requested column matches the cached column.
```

Author: @KlaasSL Done!

}
```

From DataMiner 10.4.0 [CU21]/10.5.0 [CU9]/10.5.12 onwards<!--RN 43457-->, it is possible to cache specific columns of a table using the optional `ColumnIDs` property. If no `ColumnIDs` property is specified, the entire table will be cached.

For example, the following configuration shows how to cache only specific columns of a table parameter:

```json

{
    "DataMinerID": 956,
    "ElementID": 1846,
    "ParameterID": 1500,
    "SubscriptionFilter": null,
    "ColumnIDs": "1512"
}
```

> [!NOTE]
>

> > - If both the `ColumnIDs` property and a `columns=` option in the `SubscriptionFilter` are specified, the `columns=` option in the filter takes priority.

Author: ```suggestion
```
